README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
```

Comment:
Activity: Approve a pull request

    In the pull request, click the Files changed tab
    Review the changes in the pull request by commenting on specific lines
    Above the changed code, click Review changes
    Type a comment summarizing your feedback on the proposed changes
    Select Approve to approve merging the changes proposed in the pull request
    Click Submit review